scenarios/centos-9/edpm_baremetal_deployment_ci.yml

```diff
@@ -1,4 +1,5 @@
 ---
+# MARIOS WAS HERE ! (triggerring cifmw-baremetal-nested-crc for testing)
```

Comment:
remove this before merge :)

Reply:
updating to do this now

Reply:
green run of the baremetal-nested-crc there https://review.rdoproject.org/zuul/build/e961169d4b544a9a9f121933d3b10879